visualization/app/codeCharta/ui/toolBar/downloadButton/downloadDialog/downloadDialog.component.html

```diff
@@ -3,6 +3,20 @@ <h2>Export File</h2>
 </mat-toolbar>
 
 <mat-dialog-content>
+	<div class="deprecation-warning-container">
+		<span
+			><h4>Deprecation warning!</h4>
+			This function is going to be replaced by Custom Views soon. For further information, please read the
+			<a
+				class="custom-views-link"
+				href="https://maibornwolff.github.io/codecharta/docs/custom-view/"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				How-To Article</a
```

Comment:
Let's describe in the documentation, that the Custom Feature is replacing the donwload button, if not already done.

Reply:
I think it would be best to create a new issue for updating the "user controls" article again to show custom configs as a stable feature.
As far as I know the download button is described there only.

Reply:
@LuisGuenther thank you for the hint, I will create one for that.

One remark: When clicking on the download button, there is a hint like "The download button will be replaced by Custom Views ... Click here for **more** information". In the linked documentation though, there is just a short hint that the feature will replace the download button. But there is no explenation why the feature is able to do that. I think it would be good to explain it, so that the user can comprehend this change.